### PR TITLE
Matth/analytics page

### PR DESCRIPTION
**Pull request by i-like-robots**
A few basic charts pulling data out of Keen using the [Keen SDK](https://github.com/keen/keen-js) and displayed with [Recharts](http://recharts.org/).

screen shot 2017-02-09 at 14 27 05
screen shot 2017-02-09 at 14 27 16
screen shot 2017-02-09 at 14 27 36

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -30,6 +33,7 @@
     "react-redux": "^5.0.2",
     "react-router": "^3.0.1",
     "react-tag-autocomplete": "^5.1.0",
+    "recharts": "^0.20.6",
```
Comment (ironsidevsquincy) [inline package.json]: aren't all these `dependencies`?

File: server/controllers/admin/analytics.js [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+		// TODO: use keen-query lib which has post-processing?
+		result
+			.sort((a, b) => {
+				if (a.result > b.result) {
+					return -1;
+				} else if (a.result < b.result) {
+					return 1;
+				} else {
+					return 0;
+				}
```
Comment (ironsidevsquincy) [inline server/controllers/admin/analytics.js]: `.sort((a, b) => b.result - a.result);`

File: package.json (hunk 1 above)
Comment (i-like-robots) [inline package.json]: They're only used on the client, so once we've got a bundle they're not useful anymore

File: server/controllers/admin/analytics.js (hunk 2 above)
Comment (i-like-robots) [inline server/controllers/admin/analytics.js]: Nice!

**Pull request merged**